src/operator/ResourcesAccountType.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  ComponentColor,
+  DropdownMenuTheme,
+  MultiSelectDropdown,
+} from '@influxdata/clockface'
+import React, {FC, useContext} from 'react'
+import {OperatorContext} from './context/operator'
+
+const ResourcesAccountType: FC = () => {
+  const {accountTypes, setAccountTypes} = useContext(OperatorContext)
+  const availableTypes: string[] = allAccountTypes
```

Comment:
Why do we need this rather than just passing in the `allAccountTypes` as the selectedOptions